changelogs/unreleased/download-wheels-in-rpm-chroot.yml

```
@@ -0,0 +1,3 @@
+description: Add option to pass pip index to rpm build
+change-type: patch
+destination-branches: [master, iso8]
```

```suggestion
destination-branches: [master]
```

I could have been more clear on this. On each of these repos, the master branch represents iso8, there is no iso8 branch until we prepare for its first release, when master will come to represent iso9.

The difference between this repo and inmanta-service-orchestrator is that we do OSS releases from the master branch (where inmanta-service-orchestrator) we only do so from named iso branches. Because of that, the next and stable branches (where you'll have to cherry-pick the commit) on this repo are iso8-next and iso8-stable while they're iso7-next and iso7-stable on the other repo.

Probably quite confusing if you're not up to date on the whole process.